dnf-docker-test/features/module-enable-2.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Use confirmation of enabling different module stream
+
+  @setup
+  Scenario: Testing repository Setup
+      Given I run steps from file "modularity-repo-1.setup"
+       When I enable repository "modularityABDE"
+        And I successfully run "dnf makecache"
+        And I successfully run "dnf module enable -y ModuleA:f26"
+        And I successfully run "dnf module enable -y ModuleB:f26"
+
+  Scenario: Enablement of a different stream must be confirmed by the user (no confirmation)
+       When I run "dnf module enable ModuleA:f27"
```

Comment:
Please do not run a command that requires a confirmation waiting for timeout of test step.

Reply:
Why? How would you suggest to test that dnf prompts the user for the confirmation?